site/gdocs/components/KeyIndicatorCollection.tsx

```diff
@@ -0,0 +1,309 @@
+                >
+                    {header}
+                </button>
+            </h3>
+            {/* mobile */}
+            {!isOpen && mobileHeader}
+            <div
+                id={contentId}
+                className="accordion-item__content"
+                style={{
+                    height: isOpen ? `${height}px` : "0px",
+                }}
+                role="region"
+                aria-labelledby={headerId}
+            >
+                <div ref={contentRef}>{children}</div>
+            </div>
+        </div>
+    )
+}
+
+function KeyIndicatorHeader({
+    block,
+    isContentVisible,
+}: {
+    block: EnrichedBlockKeyIndicator
+    isContentVisible?: boolean
+}) {
+    const { linkedChart } = useLinkedChart(block.datapageUrl)
+    const { linkedIndicator } = useLinkedIndicator(
+        linkedChart?.indicatorId ?? 0
+    )
+
+    if (!linkedChart) return null
+    if (!linkedIndicator) return null
+
+    const { queryParams } = Url.fromURL(linkedChart.resolvedUrl)
+    const tabFromQueryParams =
+        queryParams.tab && isValidGrapherTab(queryParams.tab)
+            ? queryParams.tab
+            : undefined
+    const activeTab =
+        tabFromQueryParams || linkedChart.tab || GrapherTabOption.chart
+
+    const source = capitalize(block.source || linkedIndicator.attributionShort)
+
+    return (
+        <div className="key-indicator-header">
+            <div className="key-indicator-header__left">
+                <FontAwesomeIcon
+                    icon={tabIconMap[activeTab]}
+                    className="key-indicator-header__tab-icon"
+                />
+                <div>
+                    <span className="key-indicator-header__title">
+                        {linkedIndicator.title}
+                    </span>
+                    {source && (
+                        <span className="key-indicator-header__source">
+                            {source}
+                        </span>
+                    )}
+                </div>
+            </div>
+            {!isContentVisible && (
+                <div>
+                    {/* desktop */}
+                    <FontAwesomeIcon
+                        icon={faPlus}
+                        className="key-indicator-header__icon"
+                    />
+                    {/* mobile */}
+                    <FontAwesomeIcon
+                        icon={faArrowRight}
+                        className="key-indicator-header__icon"
+                    />
+                </div>
+            )}
+        </div>
+    )
+}
+
+function KeyIndicatorLink({
+    block,
+    children,
+}: {
+    block: EnrichedBlockKeyIndicator
+    children: React.ReactNode
+}) {
+    const { linkedChart } = useLinkedChart(block.datapageUrl)
+    const { linkedIndicator } = useLinkedIndicator(
+        linkedChart?.indicatorId ?? 0
+    )
+
+    if (!linkedChart) return null
+    if (!linkedIndicator) return null
```

Comment:
True – we could also store indicator information with a linked chart. I guess I went with this approach since linked charts and indicators are conceptually different, so this feels cleaner to me. And perhaps a little more future proof in case we want to link indicators in the future that don't belong to a chart. No strong feelings though!